### ISSUE TITLE
Expand filtering available on participant analytics view

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#605]
# TL;DR
Allow researchers to filter participants by when they registered, rather than only by when they participated.

# Narrative
As a researcher, I may want to review recruitment based primarily on participants who registered to participate in my study, rather than on existing participants who participated in a given timeframe.

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Researcher can select whether to filter (pivot table) based on participating vs. registering in timeframe. 
- [ ] Tests of new feature
- [ ] QA on  new feature with researchers

# Implementation Notes
The cumulative participation/registration plots could be removed to free up room in the UI.
